libs/application/core/src/types/Application.ts

```diff
@@ -28,7 +28,7 @@ export interface ActionCardMetaData {
   }
 }
 
-export interface Application {
+export interface ApplicationModel {
```

Comment:
To have this more explicit and clear in the naming how about the calling model with attachments `ApplicationWithAttachments`?

Reply:
Yes! good idea!